delorean.cabal

```diff
@@ -19,8 +19,8 @@ library
                      , attoparsec                      >= 0.11       && < 0.14
                      , template-haskell
                      , text                            == 1.2.*
-                     , time                            >= 1.4        && < 1.6
-                     , tz                              <  0.1.1
+                     , time                            >= 1.4        && < 1.7
+                     , tz                              >=  0.1       && < 0.2
```

Comment:
Just going off the first example I looked up - https://github.com/ambiata/hydra/blob/master/hydra-daemon/hydra-daemon.cabal#L56.  Ideally this change should be compatible as to not force some mass rollout